desktop-src/direct3dhlsl/AddUint64.md

```diff
@@ -0,0 +1,61 @@
+---
+description: Unsigned add of 32-bit operand with the carry.
+nms.assetid:
+title: AddUint64
+ms.topic: reference
+ms.date: 07/10/2024
+topic_type:
+- APIRef
+- kbSyntax
+api_name:
+- AddUint64
+api_type:
+- NA
+---
+
+
+# AddUint64
+
+Unsigned add of 32-bit operand with the carry.
+
+## Syntax
+
+
+```syntax
+uint<c> AddUint64(uint<c> a, uint<c> b);
+```
+
+## Parameters
+
+| Item | Description |
+|------|-------------|
+| *a* | [in] A vector of type uint64_t (64-bit unsigned integer) as the first operand.  || *b* | [in] A second vector of type uint64_t (64-bit unsigned integer) as the second operand.  |
```

Comment:
This table (and many of the later ones) seems to have lost some new lines between rows and doesn't render correctly.

Reply:
Its rendering correctly in md, because the right number of `|`  but this is a simple fix. in the generator.